starter-bots/cplusplus/README.md

```diff
@@ -0,0 +1,24 @@
+# C++ bot for Entelect Challenge 2019
+
+This container includes components required to compile and run a c++ project. The start bot was a c++ version of the supplies javascript reference bot
```

Comment:
This should refer to the javascript starter bot, not the reference bot.

Other than that everything seems good 😄 just resolve the current merge conflict and we will merge your pull request in as soon as possible.

Reply:
Done. Cleaned up the readme as well since it is definitely not based on the reference bot anymore. :)